### PR TITLE
docs: edit readme

### PR DESCRIPTION
**Pull request by jdemaagd**

### CODE REVIEW

**Review by eulasi — DISMISSED** [review]
Updating Implementations? OK. For reference, what is the difference between extJunit and junitKtx(Lines 41,49 in AppDependencies.kt)?

the ext is junit for implementation tests (androidTest) testing Android framework (UI)
junit is for unit tests (no Android Framework dependencies)

File: buildSrc/src/main/kotlin/Version.kt [hunk 1]
```diff
@@ -6,41 +6,42 @@ object Version {
...
+    const val constraint = "2.1.4"
     // core
     const val coreKtx = "1.12.0"
     const val appcompat = "1.6.1"
     const val material = "1.11.0"
-
-    // network
-    const val gson = "2.10.1"
-    const val okhttp = "4.12.0"
-    const val retrofit = "2.9.0"
```
Comment (chrisivey2) [inline buildSrc/src/main/kotlin/Version.kt:17]: Why was these changed from gson to courtines

**Review by chrisivey2 — APPROVED** [review]

File: buildSrc/src/main/kotlin/Version.kt (hunk 1 above)
Comment (jdemaagd) [reply buildSrc/src/main/kotlin/Version.kt:17]: you have to look at the whole file gson is still there
it was probably re-organized
